backend/api_calls.py

```diff
@@ -794,6 +794,131 @@ def AutoUpdatePlayerGroups(firebase, player_id, new_player=False):
       AddPlayerGroupMappings(firebase, group_id, player_id)
 
 
+# TODO Decide how to make a life code as used up.
+def Infect(request, firebase):
+  """Infect a player via life code.
+
+  Infect a human and get 100 points.
```

Comment:
Maybe? Could be. This may likely change. For now, I'll just put it in the constants file.